gutenberg.php

```diff
@@ -27,4 +27,5 @@
 
 	// Register server-side code for individual blocks.
 	require_once dirname( __FILE__ ) . '/lib/blocks/latest-posts.php';
+	require_once dirname( __FILE__ ) . '/lib/blocks/categories.php';
```

Comment:
This was fixed with a combination of rebase and f23a5da24ba2dace39927ddfdb4e969c0c524f30.